app/models/channels/twilio.rb

```diff
@@ -19,11 +19,14 @@ class Channels::Twilio < Channel
   config_accessor :account_sid
   config_accessor :auth_token
   config_accessor :number
+  config_accessor :base_url
 
   attr_protected :guid
 
   before_create :create_guid
 
+  validates :base_url, format: URI::regexp(["http", "https"]), allow_nil: true, allow_blank: true
```

Comment:
If there's an option to allow nil (ie, default URL) but not blanks (ie, reject a blank non-default URL), I'd go for it?